Venmo/src/main/java/com/braintreepayments/api/VenmoRequest.java

```diff
@@ -8,6 +8,17 @@
     private boolean shouldVault;
     private String profileId;
 
+    private final @VenmoPaymentMethodUsage int paymentMethodUsage;
+
+    /**
+     * Request to tokenize an existing Venmo account.
```

Comment:
```suggestion
     * Request to tokenize a Venmo account.
```

Do we need the word "existing" here?

Reply:
Nope! Agreed here it's a bit verbose.